R/pipe_consistency_linter.R

```diff
@@ -0,0 +1,79 @@
+#' Pipe consistency linter
+#'
+#' Check that pipe operators are consistently used throughout a file. Optionally
+#' specify which pipe operator to use.
+#'
+#' @param pipe Which pipe operator to use (either `"%>%"` or `"|>"`). By default
```

Comment:
nit: "to use" is not specific enough IMO (who is using it?), I'd phrase it "Which pipe operator is valid"